src/pages/workspace/FeatureEnabledAccessOrNotFoundWrapper.tsx

```diff
@@ -47,6 +50,14 @@ function FeatureEnabledAccessOrNotFoundComponent(props: FeatureEnabledAccessOrNo
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [isPolicyIDInRoute, props.policyID]);
 
+    useEffect(() => {
+        if (props.policy?.pendingFields?.[props.featureName] === 'update' && !isOffline) {
+            return;
+        }
+        setIsPolicyFeatureEnabled(PolicyUtils.isPolicyFeatureEnabled(props.policy, props.featureName));
```

Comment:
NAB, we should create a variable for `PolicyUtils.isPolicyFeatureEnabled(props.policy, props.featureName)` instead of calling it everytime we need its value.